auxiliary/producers/replayer/replayer.go

```diff
@@ -111,19 +111,13 @@ func (r Replayer) produce() error {
 func (r Replayer) Start(log *logrus.Logger) error {
 	r.log = log.WithField("source", r.SourceName)
 	r.log.Info("Starting to produce")
-	var err error
-	r.producer, err = kafkaproducer.KafkaProducer{}.ForBroker(r.Brokers)
```

Comment:
don't we need a producer anymore?

Reply:
okay I see, so we start not for each replayer but for all replayers kafka producers with 205 in server.go

Reply:
Exactly